.github/workflows/wheel_linux_x86_64_cuda.yml

```diff
@@ -26,10 +26,9 @@ concurrency:
 jobs:
   set_wheel_build_matrix:
     if: |
-      github.event.pull_request.draft == false &&
-      github.event_name != 'pull_request' ||
-      contains(github.event.pull_request.labels.*.name, 'ci:build_wheels') ||
-      github.event_name == 'workflow_dispatch'
+        github.event_name != 'pull_request' ||
+        (github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'ci:build_wheels')) ||
+        github.event_name == 'workflow_dispatch'
```

Comment:
# Suggestion
There is no need for
```
github.event_name == 'workflow_dispatch'
```
where we already have
```
github.event_name != 'pull_request'
```
```suggestion
        github.event_name != 'pull_request' ||
        (github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'ci:build_wheels'))
```

Reply:
Removed here and from the other wheel build checks. This was an existing condition, not sure if we are missing an edge case here but agree that having both conditions is unnecessary.